app/client/src/components/pages/Community.Tabs.Protect.js

```diff
@@ -571,7 +571,7 @@ function Protect() {
                         <div css={modifiedInfoBoxStyles}>
                           <p>
                             No Watershed Health Score data available for the{' '}
-                            <em>{watershed}</em> watershed.
+                            <em>{watershed.name}</em> watershed.
```

Comment:
Line `586` of this file needs to be changed from `value: watershed` to `value: watershed.name`.

Reply:
Good catch!